### ISSUE TITLE
Blogging about Canada.ca and user needs

### ISSUE THREAD

**Issue by lisafast** [canada-ca/canada-ca.github.io#9]
Moving this from issue https://github.com/canada-ca/canada-ca.github.io/issues/7
 @RachelMuston wrote: 

> Like you, Laura's group is also in test mode re: whether this is the best place to blog. We were looking for the same thing - a place for the team to blog as opposed to a single individual. One question we had about this platform was what are the capacity limits of GitHub hosting?

First off, the answer is 1 GB capacity.  

@MaryBethBaker  We are going to be writing about Canada.ca - the user needs and research evidence  driving improvements. The Canada.ca look and feel works well for that because well, it's what the posts will be talking about! Will it work to share this repo for now?

**Comment by laurawesley** [comment]
I've been thinking about this a lot...what scope makes sense for a group blog? Is it only about research and data and testing? Or is about open government? Or public engagement? Or co-creating better digital engagement? 

I think my personal preference is to blog on the Open Government site for official stuff and on my personal platforms for my opinion pieces. I'd gladly recommend cross-posting or promote pieces that are about evidence-based patterns for public engagement web interfaces. 

So, bottom line for me is @lisafast you should go ahead and use what's there (or fork it and use it...whatever @RachelMuston and @MaryBethBaker suggest).

**Comment by MaryBethBaker** [comment]
Can't wait to read your blog @lisafast  :+1:

**Comment by lisafast** [comment]
Ok great thanks, we'll start here.
